project/ScalazBuild.scala

```diff
@@ -17,6 +17,7 @@ object Scalaz {
     "-Ypartial-unification",
     "-language:higherKinds",
     "-language:existentials",
+    "-language:implicitConversions",
```

Comment:
Hmmm.... okay.

Reply:
You literally need implicit conversions to even use enrichments.